7.1 Snapshot.md

```diff
@@ -0,0 +1,52 @@
+# 7.1 Snapshot
+
+## 7.1.1 Introduction
+This section defines the *Snapshot* protocol.
+
+
+## 7.1.2 Dependencies
+The *Snapshotting* protocol depends on:
+
++ Timestamps: Core module to enable pruning.
++ Tip Selection Algorithm: No messages can be referenced for the part of the Tangle that is being pruned.
++ UTXO: Transactions whose outputs are unspent, remain after the snapshot.
++ Branches: We require that the at the point of pruning only master branch and rejected branch exist.
++ Markers: We require that the part of the Tangle being pruned is not actively dependent on Markers any more. 
++ Approval weight: We require that the approval weight is concluded at a time much before messages reach the pruning age.
++ Voting: We require that any voting is concluded much before messages reach the pruning age. 
++ Data flow: The past cone check must be relaxed (???TODO)
+
+
+## 7.1.3 Proof of Inclusion and Pruning
+
+In order to keep the ledger concise data that is sufficiently far in the past can be removed. We call this process pruning. Thus, the IOTA protocol for a standard node does not attempt to keep all information back until the genesis. Rather the protocol effectively becomes a filter mechanism to proof whether a certain message was at some point in the past included in the ledger. This can be done e.g. through a Merkel tree proof. 
+
+Transactions are a special type of message, that cannot be pruned. Specifically when a transaction contains an unspent output, see [5.1. UTXO](./5.1%20UTXO.md), the transaction shall be kept when pruning the ledger. An optimization of this process is possible by reducing the message to only keeping the information about the unspent outputs, however the implementation will be more complicated, and thus is omitted in this first iteration of the protocol.
+
+## 7.1.5 Pruning dependencies
+
+We require several conditions to be fulfilled in order to allow for a smooth pruning process. Pruning is done when messages reach the age `DELTA_PRUNE`. 
+
+Since messages cannot attach to a message whose timestamp has an age difference of more than `DELTA`, see [4.3.4 R-URTS](./4.3%20Tip%20Selection%20Algorithm.md/#4.3.4%20R-URTS), and the timestamp precision is enforced through voting, see [4.2.3 General Timestamp rules](./4.2%20Timestamps.md/#4.2.3%20General%20Timestamp%20rules). Finalization of messages must be guaranteed to happen before the point of pruning. Thus, any message must be either confirmed or rejected before the elapse of the pruning age. This requires that both voting on timestamps and conflicts, as well as the accumulation of sufficient approval weight takes a time much smaller than `DELTA_PRUNE`. 
```

Comment:
yes. its the same as with IOTA1.0 . There the number of milestones are the clock. 
FPC provided such guarantees in regards to conflicts.
With AW the situation is the following: as long as the tip pool does not grow unbounded and a high percentage of mana issues messages regularly then the confirmation time will be in the seconds. 

It could make sense to introduce some Tangle time that only advances if a high % of mana has been seen. Using the Tangle time for calculating the age check would decrease risks with pruning imo.

Reply:
We can also extend the rule to not prune what is not confirmed, but for now it would make the rule more complex without much advantage.